lib/identity_job_log_subscriber.rb

```diff
@@ -9,10 +9,18 @@ def enqueue(event)
     json = default_attributes(event, job)
 
     if ex
-      json[:exception_class] = ex.class.name
-      json[:exception_message] = ex.message
-
-      error(json.to_json)
+      if duplicate_cron_error?(ex)
+        # The presence of "exception_message" flags this as an error in our alerting
+        json[:exception_class_warn] = ex.class.name
+        json[:exception_message_warn] = ex.message
```

Comment:
whoops meant to flip the order
```suggestion
        json[:exception_class_warn] = ex.class.name
        # The presence of "exception_message" flags this as an error in our alerting
        json[:exception_message_warn] = ex.message
```